images/ceph/Makefile

```diff
@@ -35,14 +35,21 @@ ifeq ($(BUILD_CONTEXT_DIR),)
 BUILD_CONTEXT_DIR := $(shell mktemp -d)
 endif
 
-# Note: as of version 1.3 of operator-sdk, the url format changed to:
-# ${OPERATOR_SDK_DL_URL}/operator-sdk_${OS}_${ARCH}
-# (see: https://sdk.operatorframework.io/docs/installation/)
+
+ifeq ($(GOARCH),amd64)
 S5CMD_ARCH = Linux-64bit
+else
```

Comment:
While this pattern is perhaps more correct (if/else/if/end/end), I'd suggest we follow the simpler pattern above on lines 20-24 (if/else/end). We just have the two arch's for upstream.

Reply:
done